src/coreclr/vm/jitinterface.h

```diff
@@ -1109,6 +1109,37 @@ class CEEInfo : public ICorJitInfo
 
 public:
 
+    struct SignatureContext
```

Comment:
This part of the change looks unrelated to the rest now.

Also, this `SignatureContext` seems to have the same purpose as the existing `SigTypeContext`. Do we need the duplication?